troubleshooting.md

```diff
@@ -0,0 +1,551 @@
+```
+
+##Check the status of these containers using 
+
+```
+#podman logs -f ipa-downloader
+
+#podman logs -f coreos-downloader
+
+```
+After the installation is complete copy the config file to the `~/.kube/config` dir to interact with the cluster
+
+```
+cp /install-dir/ocp/auth/kubeconfig  ~/.kube/config
+```
+Or 
+
+```
+export KUBECONFIG=/install-dir/ocp/auth/kubeconfig
+```
+
+Verify if all the control plane nodes are up and running 
+
+```
+oc get nodes
+NAME                                         STATUS   ROLES           AGE    VERSION
+master-0.cloud.lab.eng.bos.redhat.com   Ready    master,worker   4h   v1.16.2
+master-1.cloud.lab.eng.bos.redhat.com   Ready    master,worker   4h   v1.16.2
+master-2.cloud.lab.eng.bos.redhat.com   Ready    master,worker   4h   v1.16.2
+```
+
+There might be a scenario where the nodes can be stuck in `NotReady` state. To check the exact errors, login to the nodes and check for logs.
+
+```
+#ssh core@master-0 
+#crictl ps (look for the error in pod status if any)
+#crictl logs <pod-in-error-state>
+```
+Check for pod errors if any in the cluster in any namespaces
+
+```
+oc get namespaces 
+For Example: oc get pods -n openshift-kube-apiserver
+```
+
+## Bootstrap VM cannot boot up my nodes
+
+This is a usual problem, maybe we add a typo on the install-config.yaml file or just the privileges are not right on my user. In the worst case you have a problem with your network prerequisites regarding routing and accesses to iDrac from Ironic. 
+To check the problems here we have 3 basic containers all regarding ironic:
+    * Ironic-api
+    * Ironic-conductor
+    * Ironic-inspector
+
+The first to enter in action is the conductor which tries to perform the login and execute the action over IPMI, check the logs there in order to see if there is a problem on this side.
+
+```
+# podman logs -f ironic-conductor
+```
+
+The logs will show up, if there are any errors. 
+
+
+We can also follow the deployment process by seeing the `.openshift_install.log` which will be under `install-dir` folder
+
+```
+# ls -latr kni7/
+auth/                                 .openshift_install.log                terraform.baremetal.auto.tfvars.json  terraform.tfvars.json
+metadata.json                         .openshift_install_state.json         terraform.tfstate                     tls/
+```
+
+# Node Related Issues
+
+* Master Node in `NotReady` state.
+
+If the master nodes are in `NotReady` state, make sure the kubelet services on the master is enabled
+
+```
+oc get nodes
+NAME                                         STATUS     ROLES    AGE   VERSION
+master-0.kni7.cloud.lab.eng.bos.redhat.com   NotReady   master   67m   v1.16.2
+master-1.kni7.cloud.lab.eng.bos.redhat.com   Ready      master   67m   v1.16.2
+master-2.kni7.cloud.lab.eng.bos.redhat.com   Ready      master   67m   v1.16.2
+``` 
+Lets check the status of kubelet of `master-0` 
+
+```
+[core@master-0 ~]$ sudo systemctl status kubelet
```

Comment:
show how you ssh into this

Reply:
Added the procedure of how to SSH into the masters.